Lesson04/HW04.Birthday/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW04.Birthday
+{
+    internal class Program
+    {
+        public int year, month, day;
+        public void info()
+        {
+            Console.Write("Год = ");
+            year = int.Parse(Console.ReadLine());
+            Console.Write("Месяц = ");
+            month = int.Parse(Console.ReadLine());
+            Console.Write("День = ");
+            day = int.Parse(Console.ReadLine());
+        }
+        public int year_2, month_2, day_2;
+        public void info_t()
+        {
+            Console.Write("Год = ");
+            year_2 = int.Parse(Console.ReadLine());
+            Console.Write("Месяц = ");
+            month_2 = int.Parse(Console.ReadLine());
+            Console.Write("День = ");
+            day_2 = int.Parse(Console.ReadLine());
+        }
+        public int result_year, result_month;
+        public void Result()
+        {
+            if ((month_2 - month) > 0)
+            {
+                result_month = month_2 - month;
+                result_year = year_2 - year;
+                Console.WriteLine("Получается {0} лет и {1} месяцов ", result_year, result_month);
+            }
+            else
+            {
+                result_month = month_2 + 12 - month;
+                result_year = year_2 - 1 - year;
+                Console.WriteLine("Получается {0} лет и {1} месяцов ", result_year, result_month);
+            }
+        }
+        static void Main()
+        {
+            Program vadym = new Program();
+            Console.WriteLine("Настоящая дата рождения : ");
```

Comment:
Why Vadim? Why not Alex or Ivan? )))) Need to change on var program = new Program (); in all places

Reply:
Done